freportn/Eestyana_Naestra.lua

```diff
@@ -1,9 +1,21 @@
--- items: 18822, 9985, 18804, 6016, 13296
+-- items: 18822, 9985, 18804, 6016, 13296, 18735
 function event_say(e)
 	if(e.message:findi("hail")) then
 		e.self:Say("Welcome to the Hall of Truth!  May the will of Mithaniel Marr guide you through life.  We are glad to see that you have an interest in our ways.  Please speak with any of my priests or knights and they shall help you find your faith.");
 	elseif(e.message:findi("militia")) then
 		e.self:Say("The Freeport Militia is nothing more than a ragtag band of mercenaries and convicts. They act as the guards of this troubled city. Be well aware that they are not the honorable guards you may think they are. They are commanded by Sir Lucan D'Lere, the self-proclaimed ruler of Freeport. We of the Hall of Truth shall set this city free from the grip of the Militia one day. It is our duty as Knights of Truth.");
+	elseif(e.message:findi("trades")) then
+		e.self:Say("I thought you might be one who was interested in the various different trades, but which one would suit you? Ahh, alas, it would be better to let you decide for yourself, perhaps you would even like to master them all! That would be quite a feat. Well, lets not get ahead of ourselves, here, take this book. When you have finished reading it, ask me for the [second book], and I shall give it to you. Inside them you will find the most basic recipes for each trade. These recipes are typically used as a base for more advanced crafting, for instance, if you wished to be a smith, one would need to find some ore and smelt it into something usable. Good luck!");
+		e.other:SummonItem(51121); -- Item: Tradeskill Basics : Volume I
+	elseif(e.message:findi("second book")) then
+		e.self:Say("Here is the second volume of the book you requested, may it serve you well!");
+		e.other:SummonItem(51122); -- Item: Tradeskill Basics : Volume II
+	end
+end
+
+function event_enter(e)
+	if(e.other:HasItem(18735) == true) then
+		e.other:Message(15,"As you bask in the majesty of the Temple of Marr, an important looking individual, clad in shining metal armor turns to greet you. 'I can spot a new recruit a mile a way. Welcome to the Temple of Marr. I am Eastyana Naestra. Read the note in your inventory and hand it to me to begin your training. I wish you the best of luck.'");
```

Comment:
I believe for this to properly function, you need to define a proximity: e.g.

function event_spawn(e)
	local xloc = e.self:GetX();
	local yloc = e.self:GetY();
	eq.set_proximity(xloc - 50, xloc + 50, yloc - 50, yloc + 50);
end